releases-beta/how-to-guides/create-custom-workflow-parameters.md

```diff
@@ -0,0 +1,63 @@
+# Create Custom Workflow Parameters
+
+Custom workflow parameters allow you to define and manage key-value pairs that can be accessed during your software build and deployment process. This ensures that your release pipeline can use specific configurations for different environments, such as staging, production, or custom setups.
+
+## Generate Workflow Parameters in Your Project or Environment Config
+
+1. Go to the Project Config in your Aviator release project.
+2. In the build section, fetch and select your workflow.
+![](../../.gitbook/assets/release-build.png)
+3. In the "Additional workflow parameters", click "ADD" button to define a new workflow parameter.
+4. Enter the key (name of the variable) and the value. Instead of a custom value, you can also select a pre-defined value from the dropdown.
+   - **Key**: This should be a unique identifier for the variable (e.g., user).
+   - **Value**: This is the specific configuration or secret (e.g., Aviator).
+5. After adding all necessary variables, click Save to store the configurations.
+![](../../.gitbook/assets/release-custom-env.png)
+6. If you'd like to create custom workflow parameters for a specific environment, navigate to the Environment Config for that environment. In the "Additional workflow parameters" section, add the parameters in the same way as described above.
+![](../../.gitbook/assets/release-env-config.png)
+
+## Add the Custom Workflow Parameters to your Pipeline
+
+1. Go to your CI/CD workflow (GitHub Actions in this case).
+2. Add the variables as inputs on workflow dispatch.
+```yaml
+name: build
+on:
+  workflow_dispatch:
+    inputs:
```

Comment:
We also send include some other parameters that the workflow need to accept. Not sure if we want to include those as well.

Reply:
nvm I think it's OK not to include them.